homeassistant/components/plex/media_player.py

```diff
@@ -39,19 +42,28 @@
     CONF_SHOW_ALL_CONTROLS,
     CONF_REMOVE_UNAVAILABLE_CLIENTS,
     CONF_CLIENT_REMOVE_INTERVAL,
+    DEFAULT_PORT,
+    DEFAULT_SSL,
+    DEFAULT_VERIFY_SSL,
     DOMAIN as PLEX_DOMAIN,
     NAME_FORMAT,
     PLEX_CONFIG_FILE,
 )
 from .server import PlexServer
 
+DEFAULT_HOST = "localhost"
```

Comment:
We have default host in the sensor platform. We could move these to the package const module and import from there.